### ISSUE TITLE
DiceCoefficient for 2D  images 

### ISSUE THREAD

**Issue by Najat91** [wolny/pytorch-3dunet#19]
hi wolny 
 i used your DiceCoefficient in my pytorch code but it gives the memory location not the value of the dice 
` for batch, (images, masks) in enumerate(data_train):

        with torch.no_grad():

            images = Variable(images.cuda())

            masks = Variable(masks.cuda())

            outputs = model(images)

            loss = criterion(outputs, masks)

            preds = torch.argmax(outputs, dim=1).float()

            acc = accuracy_check_for_batch(masks.cpu(), preds.cpu(), images.size()[0])

            total_acc = total_acc + acc

            total_dice = DiceCoefficient(preds.cpu(),masks.cpu())

            total_loss = total_loss + loss.cpu().item()

    return total_acc/(batch+1), total_loss/(batch + 1),total_dice`

the result is DiceCoefficient object at 0x7f2d78fe97f0>

**Comment by wolny** [comment]
Create the functor first and then use it
```
dice = DiceCoefficient()
score = dice(preds, masks)
```
